tsl/src/bgw_policy/reorder_api.c

```diff
@@ -162,6 +162,20 @@ policy_reorder_add(PG_FUNCTION_ARGS)
 													 INTERNAL_SCHEMA_NAME,
 													 ht->fd.id);
 
+	/*
+	 * Try to see if the hypertable has a specified chunk length for the
+	 * default schedule interval
+	 */
+	dim = hyperspace_get_open_dimension(ht->space, 0);
+
+	partitioning_type = ts_dimension_get_partition_type(dim);
+	if (dim && IS_TIMESTAMP_TYPE(partitioning_type))
```

Comment:
The `dim` check is not needed since `dim` already used above. I would suggest an assertion after getting the open dimension instead.

```suggestion
	if (IS_TIMESTAMP_TYPE(partitioning_type))
```